src/main/java/org/interledger/ilp/exceptions/InterledgerQuotingProtocolException.java

```diff
@@ -2,6 +2,19 @@
 
 public class InterledgerQuotingProtocolException extends Exception {
```

Comment:
From the RFC: "Protocols built on top of ILP that define behavior for certain errors SHOULD specify the encoding format of error data."
I understand that an InterledgerException must be thrown and the data field use to add the ILQP specific data. I'm not sure at all. Maybe the JS code is a good reference point?